src/components/tabs/js/tabsDirective.js

```diff
@@ -154,7 +154,7 @@ function MdTabs () {
                   'md-scope="::tab.parent"></md-tab-item> ' +
               '<md-ink-bar></md-ink-bar> ' +
             '</md-pagination-wrapper> ' +
-            '<div class="_md-visually-hidden md-dummy-wrapper"> ' +
+            '<md-tabs-dummy-wrapper class="_md-visually-hidden md-dummy-wrapper"> ' +
```

Comment:
I was going based off of the original class name of `md-dummy-wrapper` which is still applied to the class.

This wrapper is visually hidden from the screen and is used to calculate the widths/heights so that it can properly position the real tabs. It's basically a way of doing off-screen rendering/double-buffering (in a way).

I'm happy to change the name if you have something better, but I think we should leave the classname the same in case someone is doing something with it (since we didn't mark it as private).